### ISSUE TITLE
Parallel builds?

### ISSUE THREAD

**Issue by amordahl** [Pancax/droidbench_android_projects#1]
Currently building the benchmark takes a very long time, because each builds individually. Is it possible to make it run in parallel? Does gradle support parallel builds?

**Comment by Pancax** [comment]
Yes, you can just start the gradle build script of each project. I didnt know how to do subprocesses with bash at the time but I think i know now.
